deploy/helm/kubecf/assets/operations/instance_groups/database.yaml

```diff
@@ -82,10 +82,23 @@
 - type: replace
   path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaadb/roles/name=uaa/name
   value: {{ .Values.features.external_database.databases.uaa.username | quote }}
+{{ if .Values.features.external_database.ca_cert -}}
+- type: replace
+  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaa?/ca_certs/0
+  value: {{ toYaml .Values.features.external_database.ca_cert | indent 2 }}
+{{- else }}
 - type: remove
   path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaa?/ca_certs
-- type: remove
+{{- end }}
+{{ if .Values.features.external_database.require_ssl -}}
+- type: replace
   path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaadb/tls_enabled?
```

Comment:
The property to enable/disable TLS for uaadb is just `tls`: https://bosh.io/jobs/uaa?source=github.com/cloudfoundry/uaa-release&version=74.12.0#p%3duaadb.tls.

Reply:
Good catch - looks like it changed from `tls_enabled` to just `tls` in v73.x https://github.com/cloudfoundry/uaa-release/commit/69dbfac3b47b774ceed7165cfa79bdaad2c6fa79